src/app/pages/OnDemandTvPage/__snapshots__/index.test.jsx.snap

```diff
@@ -433,7 +433,7 @@ exports[`should show the expired content message if episode is expired 1`] = `
   font-family: "BBC Nassim Pashto",Arial,Verdana,Geneva,Helvetica,sans-serif;
   font-weight: 700;
   font-style: normal;
-  color: #3F3F42;
+  color: #F2F2F2;
```

Comment:
Tests might not be picking up darkMode now since they are showing a change from dark to light

Reply:
Think it's the opposite - the snapshots were using detecting light mode before - now they are detecting dark mode

The font colour is supposed to be light when in dark mode